tests/test_utils.py

```diff
@@ -357,3 +358,57 @@ def test_extract_private_url_empty() -> None:
     body = """<p>Test content</p> """
     result = utils.extract_private(body)
     assert result is None
+
+
+def test_consecutive_work_days():
+    result = utils.business_day_range(
+        dateutil.parser.parse("March 17 2023"), dateutil.parser.parse("March 13th 2023")
+    )
+    assert result == 4.0, f"Dates did not match, actual result: {result}"
+
+
+def test_work_days_with_weekend():
+    result = utils.business_day_range(
+        dateutil.parser.parse("March 17 2023"), dateutil.parser.parse("March 8th 2023")
+    )
+    assert result == 7.0, f"Dates did not match, actual result: {result}"
+
+
+def test_work_days_full_month():
+    result = utils.business_day_range(
+        dateutil.parser.parse("February 28th 2023"),
+        dateutil.parser.parse("February 1st 2023"),
+    )
+    assert result == 19.0, f"Dates did not match, actual result: {result}"
+
+
+def test_work_days_start_on_weekend():
+    result = utils.business_day_range(
+        dateutil.parser.parse("March 10th 2023"),
+        dateutil.parser.parse("March 5th 2023"),
+    )
+    assert result == 5.0, f"Dates did not match, actual result: {result}"
```

Comment:
You do not need the report the actual result in the assertion message. If the assertion fails, the error will show the values. The message here could be something like "failed when the date starts on a weekend".